zerver/lib/user_groups.py

```diff
@@ -59,6 +59,7 @@ class UserGroupDict(TypedDict):
     can_join_group: int | AnonymousSettingGroupDict
     can_manage_group: int | AnonymousSettingGroupDict
     can_mention_group: int | AnonymousSettingGroupDict
+    can_leave_group: int | AnonymousSettingGroupDict
```

Comment:
Why isn't this next to `can_join_group`, which is alphabetically adjacent and also thematically adjacent?

Reply:
I messed up my alphabet to be honest, and forgot L comes after M and not before 😅 . I'll fix it wherever applicable.